### PR TITLE
Only run author flush tests when a flush agent name is defined

### PR DESCRIPTION
**Pull request by bellackn**
Now that there's a non-empty default for the flush target variable in the AEM definitions, we must update this check. https://github.com/wcm-io-devops/conga-aem-definitions/blob/develop/conga-aem-definitions/src/main/roles/aem-cms.yaml#L202-L206

### CODE REVIEW

File: tasks/aem-author-tests.yml [hunk 1]
```diff
@@ -14,4 +14,4 @@
     port: "{{ conga_config.quickstart.port }}"
     flush_target: "{{ conga_config.replication.author.flushTarget }}"
     type: "author"
-  when: conga_config.replication.author.flushTarget
+  when: conga_config.replication.author.flushTarget.name
```
Comment (tobias-richter) [inline tasks/aem-author-tests.yml]: Maybe we should test for `conga_config.replication.author.flushTarget`  and `conga_config.replication.author.flushTarget.name` here?

**Review by tobias-richter — COMMENTED** [review]
See my comments :)

File: tasks/aem-author-tests.yml (hunk 1 above)
Comment (bellackn) [reply tasks/aem-author-tests.yml]: Why? The included tasks are completely dependent on `conga_config.replication.author.flushTarget.name`, nothing else is being used in there. If this one is defined, it's clear that `conga_config.replication.author.flushTarget` exists, too. Besides that, nothing but `conga_config.replication.author.flushTarget.name` is used in the included tasks

Comment (bellackn) [reply tasks/aem-author-tests.yml]: Ah, I just saw why: line 15. ;) I will add this.

**Review by tobias-richter — APPROVED** [review]
LGTM

**Pull request merged**